hudi-client/hudi-java-client/src/test/java/org/apache/hudi/client/TestJavaHoodieBackedMetadata.java

```diff
@@ -119,7 +120,6 @@
 import org.junit.jupiter.params.provider.EnumSource;
 import org.junit.jupiter.params.provider.MethodSource;
 import org.junit.jupiter.params.provider.ValueSource;
-import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
```

Comment:
Should we remove the `LoggerFactory` as well?

Reply:
Good catch! Thought i removed all of them T.T.
Addressed!